lunatrace/bsl/hasura/migrations/lunatrace/1652400983913_add-pr-comment-disable/up.sql

```diff
@@ -1,60 +1,14 @@
-ALTER TABLE public.settings ADD COLUMN pr_feedback_enabled boolean DEFAULT true NOT NULL;
 ALTER TABLE public.settings DROP COLUMN is_org_settings;
-ALTER TABLE public.projects ADD CONSTRAINT project_settings_fkey FOREIGN KEY (settings_id) REFERENCES public.settings;
-ALTER TABLE public.organizations ADD CONSTRAINT organization_settings_fkey FOREIGN KEY (settings_id) REFERENCES public.settings;
+ALTER TABLE projects DROP COLUMN settings_id;
+ALTER TABLE organizations DROP COLUMN settings_id;
 
---  get caught up on the missing settings rows
-DO $$
-    declare
-        project record;
-    BEGIN
-FOR project IN SELECT * FROM public.projects WHERE settings_id IS NULL
-LOOP
-    WITH setting AS (INSERT INTO public.settings DEFAULT VALUES RETURNING id)
-    UPDATE public.projects
-    SET settings_id=setting.id
-    FROM setting
-    WHERE public.projects.id = project.id;
-END LOOP;
-END; $$;
+ALTER TABLE public.settings ADD COLUMN pr_feedback_disabled boolean DEFAULT false;
 
-DO $$
-    declare
-        organization record;
-    BEGIN
-        FOR organization IN SELECT * FROM public.organizations WHERE settings_id IS NULL
-            LOOP
-                WITH setting AS (INSERT INTO public.settings DEFAULT VALUES RETURNING id)
-                UPDATE public.organizations
-                SET settings_id=setting.id
-                FROM setting
-                WHERE public.organizations.id = organization.id;
-            END LOOP;
-    END; $$;
+ALTER TABLE public.settings ADD COLUMN "project_id" uuid null;
+ALTER TABLE public.settings ADD CONSTRAINT "settings_project_id_key" UNIQUE ("project_id");
 
-
--- make sure any new projects or orgs have a settings table associated
-CREATE OR REPLACE FUNCTION always_create_settings()
-    RETURNS trigger AS
-$BODY$
-BEGIN
-    if (NEW.settings_id IS NULL) then
-        INSERT INTO public.settings DEFAULT VALUES RETURNING id INTO NEW.settings_id;
-    end if;
-    RETURN NEW;
-END;
```

Comment:
We all need to get comfortable with writing SQL, if we are going to keep Hasura.  This is the tech stack we chose.  We can't on one hand say we can fix the issues with hasura by "just write the SQL" and then be too wary of SQL to actually write SQL and trust it to work reliably.  I don't love it either, but this is truly fine, normal database stuff.